cassdegrees/static/js/rules.js

```diff
@@ -194,6 +195,73 @@ Vue.component('rule_course', {
     template: '#courseRequirementTemplate'
 });
 
+Vue.component('rule_subject_area_specific', {
```

Comment:
```suggestion
Vue.component('rule_subject_area', {
```

Remove use of 'specific' as described above